### ISSUE TITLE
Respect SPI_GETCLIENTAREAANIMATION (SystemParemetersInfo)

### ISSUE THREAD

**Issue by fredemmott** [fredemmott/FUI#15]
This should be considered an accessibility problem. This probably only needs handling in `Widget::StyleTransitions`
